projects/Mallard/src/helpers/settings/resolvers.ts

```diff
@@ -43,13 +43,20 @@ const ALL_NAMES = [
  */
 export const ALL_SETTINGS_FRAGMENT = makeFragment(ALL_NAMES)
 
+export const SettingValues: Map<string, Promise<unknown>> = new Map()
```

Comment:
This in effect acts as an in-memory cache. Does it need to care about getting busted upon a setting changing?

Reply:
correct, it is being updated inside `setters.ts`. The expectation is that nothing should call `storeSetting` directly, which would not trigger logic dependent on it.

Reply:
It think, to this end, could we move `storeSetting` to `settings.ts` and ensure it's not exported. I think this will make this more explicit in future.

Reply:
@RichieAHB  you mean move it to `setters.ts` no? I'll do that now

Reply:
Yup sorry - typo! Nice thanks, other than that looks good.

Reply:
actually I'll send a separate PR because the best fix is to have get/set at the same place so that it's always kept in sync.